### PR TITLE
Added lightbar to UI

### PR DESCRIPTION
**Pull request by maefromm**
Changes are:
- Added the lightbar to UI  (Related to JIRA CAR2-161 task)
- Removed some unused code and comments. 

Tested these changes on the local development environment.

### CODE REVIEW

File: website/scripts/rosbridge.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ var t_nav_sat_fix = '';
 var t_robot_status = '';
 var t_cmd_speed = '';
 var t_lateral_control_driver = '';
+var t_light_bar_status = '/control/light_bar_status'; //02/2019: added to display lightbar on UI
```
Comment (msmcconnell) [inline website/scripts/rosbridge.js]: should be base topic name "control/light_bar_status" without the starting "/"

Comment (maefromm) [reply website/scripts/rosbridge.js]: Tested on the vehicles and local environment, it works with the slash, "/control/light_bar_status".

Comment (msmcconnell) [reply website/scripts/rosbridge.js]: What was the remapping you used?
Having a "/" in the front makes this a global namespace meaning out wrapper namespace like saxton_cav/control/light_bar_status might not be applied. All the other topics in this file are missing the "/"

**Comment by msmcconnell** [comment]
Please review the detected SonarCloud issue as well
https://sonarcloud.io/project/issues?id=usdot-fhwa-stol_CARMAWebUi&pullRequest=7&resolved=false

**Comment by maefromm** [comment]
@msmcconnell have updated based on our discussion. Thank you!

**Pull request merged**